NewsApp/app/src/main/java/com/yourcompany/newsapp/models/Article.kt

```diff
@@ -2,15 +2,22 @@ package com.yourcompany.newsapp.models
 
 import android.os.Parcelable
 import kotlinx.parcelize.Parcelize
+import com.squareup.moshi.Json
 @Parcelize
 data class Article(
+    @Json(name ="author")
     val author: String? = null,
+    @Json(name ="title")
     var title: String ,
-    var content: String,
+    @Json(name ="description")
     var description: String? = null,
-    var publishedAt: String ,
-    var source: Source ,
+    @Json(name ="url")
     var url: String ,
-    var urlToImage: String? = null ) :
+    @Json(name ="urlToImage")
+    var urlToImage: String? = null ,
+    @Json(name ="publishedAt")
+    var publishedAt: String ,
+    @Json(name ="content")
+    var content: String, ):
```

Comment:
Looks like `content` should be nullable.  The app is currently crashing when I select the first article in the list as the content is null. 

```suggestion
    var content: String? = null, ):
```

This is the associated error:

```
     Caused by: java.lang.NullPointerException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter content
```